openshift/all-jobs-template.yaml

```diff
@@ -0,0 +1,293 @@
+    description: "Do not resolve transitive dependencies"
+    displayName: "No transitive dependencies"
+    required: true
+  - name: THOTH_SOLVER_JOB_ID
+    description: "Identifier of a job run."
+    displayName: "Job identifier"
+    required: true
+  - name: THOTH_SOLVER_PACKAGES
+    description: "Packages to be resolved"
+    displayName: "Packages to be analyzed by solver"
+    required: true
+  - name: THOTH_SOLVER_OUTPUT
+    description: "Output to send solver results to"
+    displayName: "Solver output"
+    required: true
+  - name: THOTH_LOG_SOLVER
+    description: "Log solver actions"
+    displayName: "Log solver actions"
+    required: true
+    value: INFO
+  - name: THOTH_SOLVER_INDEXES
+    description: "A comma separated list of indexes to scan during solver run"
+    displayName: "Python Package Indexes"
+    value: ""
+
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "${THOTH_SOLVER_JOB_ID}"
+      labels:
+        app: thoth
+        component: solver
+        solver-type: solver-ubi-8-py36
+        operator: graph-sync
+        task: solver
+        mark: cleanup
+        ttl: "2h"
+    spec:
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: solver
+            solver-type: solver-ubi-8-py36
+            mark: cleanup
+            task: solver
+        spec:
+          restartPolicy: OnFailure
+          containers:
+            - name: solver-ubi-8-py36
+              env:
+                - name: THOTH_SOLVER
+                  value: solver-ubi-8-py36
+                - name: THOTH_LOG_SOLVER
+                  value: "${THOTH_LOG_SOLVER}"
+                - name: THOTH_SOLVER_NO_TRANSITIVE
+                  value: "${THOTH_SOLVER_NO_TRANSITIVE}"
+                - name: THOTH_SOLVER_PACKAGES
+                  value: "${THOTH_SOLVER_PACKAGES}"
+                - name: THOTH_SOLVER_OUTPUT
+                  value: "${THOTH_SOLVER_OUTPUT}"
+                - name: THOTH_SOLVER_INDEXES
+                  value: "${THOTH_SOLVER_INDEXES}"
+                - name: SENTRY_DSN
+                  valueFrom:
+                    secretKeyRef:
+                      name: thoth
+                      key: sentry-dsn
+              image: "${IMAGE_STREAM_REGISTRY}/${IMAGE_STREAM_NAMESPACE}/solver-ubi-8-py36:${IMAGE_STREAM_TAG}"
+              livenessProbe:
+                failureThreshold: 1
+                initialDelaySeconds: 21600
+                periodSeconds: 10
+                tcpSocket:
+                  port: 80
+              resources:
+                limits:
+                  cpu: 1
+                  memory: 1Gi
+                requests:
+                  cpu: 1
+                  memory: 1Gi
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "${THOTH_SOLVER_JOB_ID}"
+      labels:
+        app: thoth
+        component: solver
+        solver-type: solver-fedora-28-py36
```

Comment:
Maybe it would be a good idea to start with just `rhel-8.0-py36`, `fedora-30-py36` and `fedora-30-py-37`. It will be still enough to compute in couple of days (see my comment above).